1.1/debian/sdk/projectjson/hooks/post_push

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+set -e
+
+echo "Pushing additional sdk-projectjson tags"
+imageName=$DOCKER_REPO":1.1.0-sdk-projectjson"
+docker tag $IMAGE_NAME $imageName
+docker push $imageName
+
+imageName=$DOCKER_REPO":sdk"
+docker tag $IMAGE_NAME $imageName
+docker push $imageName
+
+imageName=$DOCKER_REPO":latest"
+docker tag $IMAGE_NAME $imageName
+docker push $imageName
```

Comment:
Currently the 1.0/debian/sdk/projectjson Dockerfile is tagged as latest in dotnet-nightly.  However, the README says that 1.1 is latest, and the dotnet repo uses 1.1 as latest, so I'm assuming that's a bug that we hadn't caught, which this will fix.